srcpkgs/python3-lxml_html_clean/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-lxml_html_clean'
+pkgname=python3-lxml_html_clean
+version=0.4.1
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-lxml"
+short_desc="HTML cleaning support for lxml"
+maintainer="Maeve Sproule <code@sprock.dev>"
+license="BSD-3-Clause"
+homepage="https://github.com/fedora-python/lxml_html_clean"
+changelog="https://raw.githubusercontent.com/fedora-python/lxml_html_clean/main/CHANGES.rst"
+distfiles="https://github.com/fedora-python/lxml_html_clean/archive/refs/tags/${version}.tar.gz"
+checksum=e66845597d26ca8e44def24535de33ab494a0875fb5746e5fcbf606461279bcb
+make_check=no # needs unpackaged memory-profiler
```

Comment:
Does this unpackaged dependency prevent collecting tests, or just a failure in running the tests? If the latter, skipping only the impacted tests is preferred.